src/app/pages/questions/components/question/text-input/text-input.component.ts

```diff
@@ -15,36 +15,53 @@ export class TextInputComponent implements OnInit {
 
   showDatePicker: boolean
   showTimePicker: boolean
+  showDurationPicker: boolean
+  showTextInput
+
   showSeconds: boolean
   dateFormat: string
+  datePickerValues: string[]
   timeFormat: string
-  customMonthNames = []
-  datetimeValue: { [key: string]: any } = {}
+  timePickerValues: string[]
+  durationPickerValues: string[]
+
+  value = {}
 
   constructor(private localization: LocalizationService) {}
 
   ngOnInit() {
     this.showDatePicker = this.type.includes('date')
     this.showTimePicker = this.type.includes('time')
+    this.showDurationPicker = this.type.includes('duration')
+    this.showTextInput = this.type.includes('text')
```

Comment:
just add ` || !this.type` to the end here to make real text fields show the text input.